### ISSUE TITLE
Setting a rotation and size before rendering

### ISSUE THREAD

**Issue by Facunnduran** [daybrush/moveable#974]
## Environments
* Framework name: React
* Framework version: 18.2
* Moveable Component version: react-moveable 0.50.7
* Testable Address(optional):

## Description
So i've spent the better part of last month learning how to use the moveable package and I've managed to save the position of an element into a firestore database so than after refreshing, the element stays on the same spot. However, i'm having trouble saving and updating size and rotation on refresh. Any ideas on how i can do it? Thanks.

**Comment by daybrush** [comment]
@Facunnduran 


moveable provides only brief information of target through `.getRect()` method.
But this information is not helpful when you refresh.
Save target's cssText.

**Comment by Facunnduran** [comment]
Thanks for your answer! I'm now saving the CSS transform property into the database. However, it appears to be messing up with the position of the element and now when i refresh it renders twice as far away from the center of the screen (the begining point) than it should. Thanks anyways.

**Comment by Facunnduran** [comment]
By the way, after trying out the demo, i love how you can create a moveable element by clicking on the "Text" icon, then clicking on the board and dragging. Could you summarize how you made it? Thanks
